ow_lander/scripts/arm_action_servers.py

```diff
@@ -85,7 +85,8 @@ def now_from_start(start):
             self._update_feedback()
 
         success = trajectory_async_executer.success(
-        ) and trajectory_async_executer.wait()
+        ) and trajectory_async_executer.wait() and trajectory_async_executer.get_state() != 2
```

Comment:
Instead of '2', a named constant would make this more understandable.

Reply:
Done!

Reply:
Done!